compleet-client/src/utils.rs

```diff
@@ -15,15 +15,21 @@ pub fn echoerr<M: Display>(lua: &Lua, msg: M) -> LuaResult<()> {
 pub fn echowar<M: Display>(lua: &Lua, msg: M) -> LuaResult<()> {
     self::echo(lua, msg.to_string(), messages::WARNING_MSG_TAG)
 }
-
-// TODO: create highlight group
-/// Echoes a info message.
+///
```

Comment:
keep the `TODO`

Reply:
I'm still learning about git. What's a git way of solving this other than retyping the whole text?

Reply:
Do I add the `TODO` message to `echowar` and `echoerr` functions as well?

Reply:
nope